dist/index.html

```diff
@@ -0,0 +1 @@
+<!doctype html><html lang="en"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"><title>Product cards</title><link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"><link rel="stylesheet" href="index.9da5a695.css"></head><body class="page"> <div class="card" data-qa="card"> <img src="imac.202c52ff.jpeg" alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)" class="card__image"> <p class="card__title"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A) </p> <p class="card__code"> Product code: 195434 </p> <div class="card__reviews"> <div class="stars"> <div class="stars__star"></div> <div class="stars__star"></div> <div class="stars__star"></div> <div class="stars__star"></div> <div class="stars__star"></div> </div> <p class="card__reviews card__reviews--value">Reviews: 5</p> </div> <div class="card__price"> <p class="card__price card__price--text">Price:</p> <p class="card__price card__price--value">$2,199</p> </div> <div class="card__button" data-qa="hover"> BUY </div> </div> </body></html>
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use `git add src/.` & `git commit -m 'your message'`